torch/_dynamo/variables/builtin.py

```diff
@@ -1719,6 +1719,8 @@ def call_getattr(
                         if source is not None:
                             return VariableBuilder(tx, source)(base)
                         return SourcelessBuilder.create(tx, base)
+                    if name == "__flags__":
+                        return ConstantVariable.create(value.__flags__)
```

Comment:
is this guaranteed to be constant?  do we need to guard on it?

Reply:
It is a readonly attribute:

```python
In [1]: class Foo:
   ...:     pass
   ...:     

In [2]: Foo.__flags__
Out[2]: 22040

In [3]: Foo.__flags__ = 1
AttributeError: readonly attribute
```

Reply:
Can we assert it's a read only attribute somehow?